src/linear_solver_logs/solve_log_full_ma.jl

```diff
@@ -0,0 +1,352 @@
+# This file is part of RLinearAlgebra.jl
+
+# using LinearAlgebra
+"""
+    MAInfo
+
+A mutable structure that stores information relevant to the moving average of the progress estimator.
+
+# Fields
+- `lambda1::Int64`, the width of the moving average during the fast convergence phase of the algorithm,
+  this has a default value of one.    
+- `lambda2::Int64`, the width of the moving average in the slower convergence phase, this has a default value of 30.
+- `lambda::Int64`, the value of the moving average width at the current iteration.
```

Comment:
These variables need more description and/or this needs a reference.

Reply:
I have added more description and a reference to both papers.